aten/src/ATen/native/Loss.cpp

```diff
@@ -296,7 +296,8 @@ Tensor soft_margin_loss(
 }
 
 Tensor smooth_l1_loss(const Tensor& input, const Tensor& target, const int64_t reduction, double beta) {
-  if (beta <= 0)
+  TORCH_CHECK(beta >= 0, "smooth_l1_loss does not support negative values for beta.")
+  if (beta == 0)
```

Comment:
nit: we usually surround with `{}` or put the call on the same line.  It's too easy to assume indent scope means something.